.github/workflows/test.yml

```diff
@@ -200,8 +200,7 @@ jobs:
       - name: Run unit tests
         env:
           GC_DONT_GC: '1'
-        run: nix build .#hs-backend-booster:test:unit-tests --extra-experimental-features 'nix-command flakes' --print-build-logs
-
+        run: nix build .#checks.unit-tests --extra-experimental-features 'nix-command flakes' --print-build-logs
```

Comment:
While inspecting the integration test log output, I found that we are not actually _running_ the unit tests, we are just _building them_. However, this invocation clearly does not run them either, it fails.